dask/tests/test_core.py

```diff
@@ -45,6 +46,27 @@ def test_preorder_traversal():
     assert list(preorder_traversal(t)) == [add, sum, list, 1, 2, 3]
 
 
+@pytest.fixture(scope="class")
+def mydask(request):
+    d = {':x': 1,
+         ':y': (inc, ':x'),
+         ':z': (add, ':x', ':y')}
+    # set a class attribute on the invoking test context
+    request.cls.mydask = d
```

Comment:
If using `unittest.TestCase` (which I think is appropriate here) then it seems wise to just stick to that and not mix in pytest stuff.  I would use the `setUp` method here instead of a pytest fixture.